README.rst

```diff
@@ -15,7 +15,7 @@ Python Client for Service Directory
 .. |versions| image:: https://img.shields.io/pypi/pyversions/google-cloud-service-directory.svg
    :target: https://pypi.org/project/google-cloud-service-directory/
 .. _Cloud Service Directory API: https://cloud.google.com/service-directory
-.. _Client Library Documentation: https://googleapis.dev/python/recommendationengine/latest
+.. _Client Library Documentation: https://googleapis.dev/python/servicedirectory/latest/index.html
```

Comment:
Do you need the explicit "/index.html"? It WfM w/just https://googleapis.dev/python/servicedirectory/latest